### PR TITLE
make timestamp numeric

### PR DESCRIPTION
**Pull request by mangas**

### CODE REVIEW

**Review by evaporei — COMMENTED** [review]
LGTM, but since this is a Scalar value change, perhaps there's something missing, I would be more confident if @lutter takes a look too

File: store/postgres/src/chain_store.rs [hunk 1]
```diff
@@ -617,12 +617,30 @@ mod data {
...
 
+            fn try_to_int(ts: Option<String>) -> Result<Option<u64>, StoreError> {
+                let ts = match ts {
+                    Some(str) => str,
+                    None => return Ok(None),
+                };
+
+                let radix = if ts.starts_with("0x") { 16 } else { 10 };
+
+                u64::from_str_radix(&ts[2..], radix)
```
Comment (lutter) [inline store/postgres/src/chain_store.rs:628]: This isn't right when the string is base 10 - there we don't want to skip the first two chars

File: graphql/src/schema/meta.graphql [hunk 2]
```diff
@@ -55,8 +55,8 @@ type _Block_ {
   hash: Bytes
   "The block number"
   number: Int!
-  "Timestamp of the block if available, format depends on the chain" 
-  timestamp: String
+  "Timestamp of the block if available" 
+  timestamp: Int
```
Comment (lutter) [inline graphql/src/schema/meta.graphql:59]: The comment should also say something about how to interpret that number - maybe add `Integer representation of the timestamp stored in blocks for the chain` to tell people that we don't do any other processing

**Review by lutter — APPROVED** [review]
Looks good - two comments that should be addressed before merging, but other than that ok.

File: graphql/src/schema/meta.graphql [hunk 3]
```diff
@@ -55,8 +55,8 @@ type _Block_ {
   hash: Bytes
   "The block number"
   number: Int!
-  "Timestamp of the block if available, format depends on the chain" 
-  timestamp: String
+  "Integer representation of the timestamp store in blocks for the chain"
```
Comment (mangas) [inline graphql/src/schema/meta.graphql:58]:
```suggestion
  "Integer representation of the timestamp stored in blocks for the chain" 
```

**Pull request merged**